dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jwt1.0.internal.ee-6.0.feature

```diff
@@ -4,7 +4,7 @@ singleton=true
 WLP-DisableAllFeatures-OnConflict: false
 visibility = private
 -features=\
-  com.ibm.websphere.appserver.servlet-3.0; ibm.tolerates:="3.1, 4.0"
+  io.openliberty.servlet.internal-3.0; ibm.tolerates:="3.1, 4.0"
```

Comment:
I'm not sure I understand how we are doing this for jwt-1.0 without breaking zero migration.

Reply:
Since 3.0, 3.1 and 4.0 bring in the servlet 3.0 public feature, zero migration is not broken.